### PR TITLE
Remove old code which uses pwd

### PR DESCRIPTION
**Pull request by daniviga**
pwd is not portable on Windows

### CODE REVIEW

**Comment by micheles** [comment]
You will probably break some test, make sure to run them on Jenkins

**Comment by micheles** [comment]
Also, remember to update the changelog!

**Comment by daniviga** [comment]
Tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1830/ but I only changed the error message now

File: openquake/engine/config.py [hunk 1]
```diff
@@ -143,10 +142,8 @@ def abort_if_no_config_available():
         msg = (
             "\nYou are not authorized to read any of the OpenQuake "
             "configuration files.\n"
-            "Please contact a system administrator or run the following "
-            "command:\n\n"
-            "   sudo gpasswd --add %s openquake"
-            % pwd.getpwuid(os.geteuid()).pw_name)
+            "Please check permissions on the configration files in %s."
+            % cfg._get_paths())
```
Comment (micheles) [inline openquake/engine/config.py]: While you are at it, you could fix the mispelling `configration` -> `configuration`

**Comment by micheles** [comment]
LGTM

**Pull request merged**